Dockerfile.tailscale

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:latest
+
+RUN apt-get update \
+    && apt-get install -y gnupg curl \
+    && curl -fsSL https://pkgs.tailscale.com/stable/ubuntu/focal.gpg | apt-key add - \
+    && curl -fsSL https://pkgs.tailscale.com/stable/ubuntu/focal.list | tee /etc/apt/sources.list.d/tailscale.list \
+    && apt-get update \
+    && apt-get install -y tailscale \
```

Comment:
Um. We could have here an `ARG` to do integration tests with multiple Tailscale versions, right?

Reply:
Yes, I would argue that could be an improvement for when it is hooked up with ci, so we can do a matrix